package.json

```diff
@@ -20,24 +19,39 @@
     "moment": "^2.19.3",
     "moment-timezone": "^0.5.27",
     "pickadate": "^3.5.6",
+    "prop-types": "^15.7.2",
```

Comment:
Do you need prop-types?  Looked like you were using TS.

Reply:
Ah yeah, I think I added that as I was following the initial setup from https://github.com/CruGlobal/give-web/commits/react that Bill shared with me. I'll remove it.